build_daemon/lib/constants.dart

```diff
@@ -10,6 +10,16 @@ const readyToConnectLog = 'READY TO CONNECT';
 const versionSkew = 'DIFFERENT RUNNING VERSION';
 const optionsSkew = 'DIFFERENT OPTIONS';
 
+/// These are used when serializing log messages over stdout.
+///
+/// Serialized logs must be preceded with the [logStartMarker] on its own line,
+/// and terminated with a [logEndMarker] also on its own line.
+///
+/// This allows multi-line logs to be sanely serialized via stdout, and mixed
+/// with other generic messages.
+const logStartMarker = 'BUILD DAEMON LOG START';
```

Comment:
[nit] do we want something shorter?

Reply:
Only a few logs actually are  handled this way, we went for uniqueness over efficiency here for now.